src/event-stream/event-server.ts

```diff
@@ -722,111 +738,126 @@ export async function startEventServer(opts: {
-        if (payload.length > 10_000_000) {
-          payload = 'payload body too large for logging';
-        }
-        logger.debug(`[stacks-node event] ${eventPath} ${payload}`);
-      }
-      next();
-    })
-  );
-
   app.post(
     '/new_block',
-    asyncHandler(async (req, res) => {
+    asyncHandler(async (req, res, next) => {
       try {
         const msg: CoreNodeBlockMessage = req.body;
         await messageHandler.handleBlockMessage(opts.chainId, msg, db);
         res.status(200).json({ result: 'ok' });
+        next();
       } catch (error) {
         logError(`error processing core-node /new_block: ${error}`, error);
         res.status(500).json({ error: error });
       }
-    })
+    }),
+    handleRawEventRequest
   );
 
   app.post(
     '/new_burn_block',
-    asyncHandler(async (req, res) => {
+    asyncHandler(async (req, res, next) => {
       try {
         const msg: CoreNodeBurnBlockMessage = req.body;
         await messageHandler.handleBurnBlock(msg, db);
         res.status(200).json({ result: 'ok' });
+        next();
       } catch (error) {
         logError(`error processing core-node /new_burn_block: ${error}`, error);
         res.status(500).json({ error: error });
       }
-    })
+    }),
+    handleRawEventRequest
   );
 
   app.post(
     '/new_mempool_tx',
-    asyncHandler(async (req, res) => {
+    asyncHandler(async (req, res, next) => {
       try {
         const rawTxs: string[] = req.body;
         await messageHandler.handleMempoolTxs(rawTxs, db);
         res.status(200).json({ result: 'ok' });
+        next();
       } catch (error) {
         logError(`error processing core-node /new_mempool_tx: ${error}`, error);
         res.status(500).json({ error: error });
       }
-    })
+    }),
+    handleRawEventRequest
   );
 
   app.post(
     '/drop_mempool_tx',
-    asyncHandler(async (req, res) => {
+    asyncHandler(async (req, res, next) => {
       try {
         const msg: CoreNodeDropMempoolTxMessage = req.body;
         await messageHandler.handleDroppedMempoolTxs(msg, db);
         res.status(200).json({ result: 'ok' });
+        next();
       } catch (error) {
         logError(`error processing core-node /drop_mempool_tx: ${error}`, error);
         res.status(500).json({ error: error });
       }
-    })
+    }),
+    handleRawEventRequest
   );
 
   app.post(
     '/attachments/new',
-    asyncHandler(async (req, res) => {
+    asyncHandler(async (req, res, next) => {
       try {
         const msg: CoreNodeAttachmentMessage[] = req.body;
         await messageHandler.handleNewAttachment(msg, db);
         res.status(200).json({ result: 'ok' });
+        next();
       } catch (error) {
         logError(`error processing core-node /attachments/new: ${error}`, error);
         res.status(500).json({ error: error });
       }
-    })
+    }),
+    handleRawEventRequest
```

Comment:
It looks like branch https://github.com/hirosystems/stacks-blockchain-api/pull/1359 was merged into this branch, I'm assuming that was unintentional? It looks like a rebase or cherrypicking is necessary for this branch